ifadapter/controller/controller_error.go

```diff
@@ -0,0 +1,13 @@
+package controller
+
+type unKnownReferenceTypeError struct{}
```

Comment:
"unknown" is one word, no need to split.

```suggestion
type unknownReferenceTypeError struct{}
```

Reply:
Corrected.